### PR TITLE
Nutri calendar

### PR DESCRIPTION
**Pull request by Andree37**
It is quite big because I lost focus and started doing it all 😳

### CODE REVIEW

File: src/components/Calendars/NutritionistCalendar.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        let firstDayNextMonth = add(firstDayCurrentMonth, { months: -1 });
+        setCurrentMonth(format(firstDayNextMonth, 'MMM-yyyy'));
+    }
+
+    function nextMonth() {
+        let firstDayNextMonth = add(firstDayCurrentMonth, { months: 1 });
+        setCurrentMonth(format(firstDayNextMonth, 'MMM-yyyy'));
+    }
+
+    let selectedDayMeetings = appointments.filter((consultation) => isSameDay(parseISO(consultation.attributes.date), selectedDay));
```
Comment (tommasini) [inline src/components/Calendars/NutritionistCalendar.tsx]: Why using let here?

File: src/components/Calendars/NutritionistCalendar.tsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                            <div>T</div>
+                            <div>F</div>
+                            <div>S</div>
+                        </div>
+                        <div className="grid grid-cols-7 mt-2 text-sm">
+                            {days.map((day: Date, dayIdx: number) => (
+                                <div key={day.toString()} className={`${dayIdx === 0 && colStartClasses[getDay(day)]} py-1.5`}>
+                                    <button type="button" onClick={() => setSelectedDay(day)} className={`${isEqual(day, selectedDay) && 'text-white'} ${!isEqual(day, selectedDay) && isToday(day) && 'text-red-500'} ${!isEqual(day, selectedDay) && !isToday(day) && isSameMonth(day, firstDayCurrentMonth) && 'text-gray-900'} ${!isEqual(day, selectedDay) && !isToday(day) && !isSameMonth(day, firstDayCurrentMonth) && 'text-gray-400'} ${isEqual(day, selectedDay) && isToday(day) && 'bg-red-500'} ${isEqual(day, selectedDay) && !isToday(day) && 'bg-gray-900'} ${!isEqual(day, selectedDay) && 'hover:bg-gray-200'} ${(isEqual(day, selectedDay) || isToday(day)) && 'font-semibold'} mx-auto flex h-8 w-8 items-center justify-center rounded-full`}>
```
Comment (tommasini) [inline src/components/Calendars/NutritionistCalendar.tsx:77]: Wtf is happening on this button? XD

Comment (tommasini) [reply src/components/Calendars/NutritionistCalendar.tsx:77]: Please can we abstract this to a function, give a name for be understandable ? <3

**Review by tommasini — APPROVED** [review]
LGTM! But please address the comments first

File: src/components/Calendars/NutritionistCalendar.tsx (hunk 1 above)
Comment (Andree37) [reply src/components/Calendars/NutritionistCalendar.tsx]: you are right, I was putting let on everything, changed to const now

**Review by tommasini — APPROVED** [review]
AMAZING LGTM!

**Pull request merged**